backends/graphics/surfacesdl/surfacesdl-graphics.h

```diff
@@ -200,6 +200,9 @@ class SurfaceSdlGraphicsManager : public SdlGraphicsManager {
 
 	virtual SDL_Surface *SDL_SetVideoMode(int width, int height, int bpp, Uint32 flags);
 	virtual void SDL_UpdateRects(SDL_Surface *screen, int numrects, SDL_Rect *rects);
+	virtual int SDL_SetColors(SDL_Surface *surface, SDL_Color *colors, int firstcolor, int ncolors);
```

Comment:
Do these need to be virtual?

(Imagine the SDL compatibility hack being overriden by some port to implement some other hack)